x-pack/agent/_meta/agent.docker.yml

```diff
@@ -8,12 +8,16 @@ outputs:
     username: elastic
     password: changeme
 
-streams:
-  - type: event/file
-    enabled: true
-    paths:
-      - /var/log/hello1.log
-      - /var/log/hello2.log
+datasources:
+  - namespace: default
+    use_output: default
+    inputs:
+      - type: system/metrics
+        streams:
+          - metricset: cpu
```

Comment:
Had a quick chat with @michalpristas about this on Slack. The problem here is that this will send data to `metrics-generic-default` but it should end up in `metrics-system.cpu-default`. We probably have to add the dataset here at the moment and simplify it later.